RecaptchaValidation/Models/RecaptchaResponseMessage.cs

```diff
@@ -9,7 +9,7 @@ namespace RecaptchaValidation.Models
      * with mapping or anything like that. 
      */
     [DataContract]
-    public class RecaptchaResponseMessage: IRecaptchaResponseMessage
+    public sealed class RecaptchaResponseMessage
```

Comment:
Can possibly use `internal` instead of `public`. If this class is directly exposed to the external consumers of the API, then that isn't the case though. Internal when you can get it though.

Reply:
I get a couple of these in response to trying that: 
![image](https://user-images.githubusercontent.com/43724406/223203093-537a4175-8c7e-45b0-a3b6-62c1e829f593.png)